cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -154,6 +154,18 @@
     # adding a missing bpf mount
     mount fstype=bpf options=(rw) bpf -> /sys/fs/bpf/,
 
+    # For mounting base dir by dir (write dirs and mount on them)
+    /tmp/snap.rootfs_** rw,
+    mount options=(remount ro) -> /tmp/snap.rootfs_*/,
+    mount options=(rw rbind) {/var/lib/snapd,}/snap/core**/ -> /tmp/snap.rootfs_**/,
+    # For mounting individual files
+    mount options=(rw bind) {/var/lib/snapd,}/snap/core** -> /tmp/snap.rootfs_**,
```

Comment:
It's tricky, because one rule uses `bind` while the other uses `rbind`. According to https://gitlab.com/apparmor/apparmor/-/wikis/AppArmor_Core_Policy_Reference#options-and-flags we could merge the rules using
```
    mount options in (rw, bind, rbind)...
```
but AFAIU this is not yet implemented in the apparmor parser. So I'm afraid we need to keep both lines.